util/util_test.go

```diff
@@ -6,6 +6,12 @@ import (
 	"testing"
 )
 
+func TestBinaryInstalled(t *testing.T) {
```

Comment:
Actually if we test for get availability we also need to test for osc availability. Which doenst make sense. It could be that the user only wants to have his contributions from wiki + osc and has no git contributions, right?